modules/schematics/src/store/files/__statePath__/index.ts.template

```diff
@@ -1,11 +1,11 @@
+<% if (!isLib) { %>import { isDevMode } from '@angular/core';<% } %>
 import {
```

Comment:
This will probably create a blank line. We could move the closing brace to the line below:

```suggestion
<% if (!isLib) { %>import { isDevMode } from '@angular/core';
<% } %>import {
```